Yoast/wordpress-seo
# Assign jQuery to its $ alias

## Summary

* Fixes a bug where the jQuery `$` is undefined when adding an image block in the block editor.


## Test instructions
- repeat the steps from https://github.com/Yoast/bugreports/issues/682
- see there's no error in the console any longer
- check the analysis results: these results should NOT appear:
> - Image alt attributes: No images appear on this page. Add some!
> - Text length: The text contains 1 word. This is far below the recommended minimum of 300 words. Add more content.

Instead, these results should appear (if the image has no alt attribute):
> - Text length: The text contains 0 words. This is far below the recommended minimum of 300 words. Add more content.
> - Image alt attributes: Images on this page do not have alt attributes that reflect the topic of your text. Add your keyphrase or synonyms to the alt tags of relevant images!

## UI changes

## Documentation

## Quality assurance


Fixes https://github.com/Yoast/bugreports/issues/682
